game-two/Sources/App/Shared/Scenes/AnimationController/Animations/HangingAnimation.cs

```diff
@@ -96,13 +96,14 @@ public void StartHangingAnimation()
     public void SetHangingAnimationOf()
 	{
 		GD.Print("[JumpAnimation] Set jumping off");
-		var parent = (Player) this.Tentacule.GetParent();
+		Player parent = (Player) this.Tentacule.GetParent();
```

Comment:
A vérifier : je coris que l'on peut faire : this.Tentacule.GetParent<Player>() ?

Reply:
Si c'est par rapport au cast, on à pas le choix si on ne caste pas il n'arrive pas à convertir les type Node en un autre (Player, KinematicBody2D, Monstre, PixBlock, etc...)